### PR TITLE
add-remot-paper

### PR DESCRIPTION
**Pull request by YizhaoGao**
Add a paper "REMOT: A Hardware-Software Architecture for Attention-Guided Multi-Object Tracking with Dynamic Vision Sensors on FPGAs"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -473,7 +473,8 @@ IEEE Conf. Computer Vision and Pattern Recognition Workshops (CVPRW), 2021. [You
 55th Annual Conf. on Information Sciences and Systems (CISS), 2021.
 - [Alzugaray, I., Ph.D. Thesis, 2022](#Alzugaray22PhD),  
 *Event-driven Feature Detection and Tracking for Visual SLAM*.
-
+- [Gao, et at., FPGA, 2022](#GAO22FPGA),
```
Comment (guillermogb) [inline README.md:476]: Hello. This work is not doing corner detection and tracking... so it should not be in this subcategory; it should be in the upper category "Feature Detection and Tracking". Could you move it, please?
Also, please check the key value works: Gao22fpga (case sensitive?).

**Review by guillermogb — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -425,7 +425,8 @@ Int. Conf. on Machine Vision and Applications (ICMVA), 2021. [Code](https://gith
 IEEE Int. Conf. Computer Vision Workshop (ICCVW), 2021. [Youtube](https://youtu.be/ST6Z-3SlNS4), [Code](https://github.com/anindya2001/GSCEventMOD).
 - [Alzugaray, I., Ph.D. Thesis, 2022](#Alzugaray22PhD),  
 *Event-driven Feature Detection and Tracking for Visual SLAM*.
-
+- [Gao, et at., FPGA, 2022](#Gao22FPGA),
```
Comment (guillermogb) [inline README.md:428]: Is this a secondary entry? Where is the main one?

**Review by guillermogb — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (YizhaoGao) [reply README.md:428]: Under the event processor and platform

Comment (guillermogb) [reply README.md:428]: ok. I lost track of it for a split second.

**Review by guillermogb — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (YizhaoGao) [reply README.md:428]: Thanks

**Pull request merged**